Cargo.toml

```diff
@@ -31,3 +31,7 @@ toml = "0.5.8"
 tracing = { version = "0", default-features = false }
 tracing-subscriber = "0"
 serde = { version = "1", default-features = false }
+
+[patch.'https://github.com/casper-network/casper-node.git']
+casper-binary-port = { path = "../casper-node/binary_port" }
```

Comment:
This won't work in CI, or in any workspace that doesn't have `../casper-node`, right?
Wouldn't it be more universal if you pushed your casper-node changes to a branch and refer casper-binary-port to a github branch?

Reply:
Updated in https://github.com/casper-network/casper-sidecar/pull/315/commits/c1af7cfd7480502ad8d7a38e3177d9f12b528fe9